pages/instructorAssessmentGroups/instructorAssessmentGroups.sql

```diff
@@ -40,11 +40,12 @@ SELECT
     u.uid
 FROM
     groups AS gr
-    JOIN group_users AS gu ON gr.group_config_id = $group_config_id AND gu.group_id = gr.id AND gr.deleted_at IS NULL
-    RIGHT JOIN enrollments AS e ON e.user_id = gu.user_id AND e.course_instance_id = $course_instance_id
+    JOIN group_users AS gu ON gu.group_id = gr.id AND gr.group_config_id = $group_config_id AND gr.deleted_at IS NULL
```

Comment:
It looks a bit weird to me to have `gr.group_config_id = $group_config_id AND gr.deleted_at IS NULL` in this join condition, given that this is a JOIN with `group_users`. Putting these things into join conditions looks like the same sort of bug that this PR is fixing.

Would it be cleaner to shift `gr.group_config_id = $group_config_id AND gr.deleted_at IS NULL` into the WHERE clause?

Reply:
I tried this earlier, but it would give a different result as I am using a RIGHT JOIN here.
An alternative way would be: 
leave gr.deleted_at at the same line and move `gr.group_config_id = $group_config_id` to WHERE:
```
AND (gr.group_config_id IS NULL
OR gr.group_config_id != $group_config_id)
```
This way is not straightforward. Which do you prefer?

Reply:
Let's leave it as is for now. I actually think that this might be much clearer if we just use an EXCEPT to compute the set difference between all enrolled students and all students in groups.